app/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# universal-search-recommendation-app
+#
+# VERSION 0.1
+
+FROM python:3.5
+MAINTAINER Chuck Harmston <chuck@mozilla.com>
```

Comment:
Might be better to point this directive at the public mailing list or the github issues URL for this repo.